### PR TITLE
Created "Slack" topic

### PR DESCRIPTION
**Pull request by Sigmale1000**
### Please confirm this pull request meets the following requirements:

- [X] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [X] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [X] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [X] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [X] My folder contains a `*.png` image (if applicable) and `index.md`
- [X] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Slack is a good messaging app getting used by a lot of dev teams and companys.

### CODE REVIEW

File: topics/slack/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: slack-messaging, slack-chat
```
Comment (smashwilson) [inline topics/slack/index.md:2]: The `slack-messaging` topic is empty, and `slack-chat` has only two repositories that look unrelated. Let's just get rid of the alias.

```suggestion
```

**Review by smashwilson — APPROVED** [review]
:+1: Thanks!

**Comment by Sigmale1000** [comment]
Sure, no problem!
